From 11a6fc258c6765a43b973b15e86fb4bce7675202 Mon Sep 17 00:00:00 2001
From: Manoj Gupta <manojgupta@google.com>
Date: Mon, 14 Oct 2019 11:37:20 -0700
Subject: [PATCH 2/2] Use regular archives.

libmodp_b64 needs to be linked by libchrome later so
use regular archives.

BUG=chromium:1012803
TEST=libchrome can link with libmodp_b64.

Change-Id: Ie0bbcc8a54051d4136463c95762ba8343e487862
---
 BUILD.gn | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/BUILD.gn b/BUILD.gn
index b2fe97d..9107609 100644
--- a/BUILD.gn
+++ b/BUILD.gn
@@ -16,6 +16,12 @@ group("all") {
 }
 
 static_library("modp_b64") {
+  configs -= [
+    "//common-mk:use_thin_archive",
+  ]
+  configs += [
+    "//common-mk:nouse_thin_archive",
+  ]
   sources = [
     "modp_b64.cc",
     "modp_b64.h",
-- 
2.23.0

